linux/network_device_ops/hot_add_network_adapter.yml

```diff
@@ -1,135 +1,169 @@
 # Copyright 2023-2024 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-# Hot add a new network adapter and apply network config
+# Add a new network adapter and apply network config
 #
 - name: "Initialize facts of the new {{ adapter_type }} network adapter"
   ansible.builtin.set_fact:
-    new_network_adapter: ""
+    new_network_adapter_name: ""
     new_network_adapter_status: ""
-    new_network_adapter_mac_addr: ""
+    new_network_adapter_mac: ""
     new_network_adapter_ip: ""
+    primary_nic_known_issue: false
 
-- name: "Clear dmesg ring buffer"
+# For SLES 15 on ESXi on ARM, if the primary network adapter type is e1000e, and the second one
+# is vmxnet3, their interface name will be swapped after power on. So here changes
+# the primary network adapter's name to "eth1", so that it can take effect after cold
+# add a new vmxnet3 adapter and VM power on.
```

Comment:
'so that it can take effect after cold add a new vmxnet3 adapter and VM power on' need to be refined.